lib/routes/routeBackbeat.js

```diff
@@ -1160,7 +1160,7 @@ function batchDelete(request, response, userInfo, log, callback) {
             }
             dataWrapper.data.delete(_loc, log, err => {
                 if (err?.is?.ObjNotFound) {
-                    log.info('batch delete: data location do not exist', {
+                    log.warn('batch delete: data location do not exist', {
```

Comment:
not sure this is a warning : it actually happens in some cases when deleting an archived object (cold).